src/collective/volto/formsupport/restapi/services/validation/configure.zcml

```diff
@@ -0,0 +1,23 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:plone="http://namespaces.plone.org/plone"
+    xmlns:zcml="http://namespaces.zope.org/zcml">
+
+  <plone:service
+    method="POST"
+    name="@validate-email-message"
```

Comment:
maybe it's better "validate-email-address"? Why message?

Reply:
Use as a verb here

Reply:
i don't get it..but we are validating an email address, not a message